tests/0117-mock_errors.c

```diff
@@ -255,6 +255,57 @@ static void do_test_offset_commit_request_timed_out(rd_bool_t auto_commit) {
         SUB_TEST_PASS();
 }
 
+/**
+ * @brief PR #4030. A COORDINATOR_LOAD_IN_PROGRESS error is returned while a
```

Comment:
It is fine referencing an issue for background information, but we typically don't reference PRs, so just remove the 'PR #4030' part.

Reply:
It is not clear what "returned" refers to here, so flesh it out to say "returned from consumer_poll()" or similar.

Reply:
Done